regulations/static/regulations/js/source/views/header/preamble-head-view.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var PreambleHeadView = Backbone.View.extend({
+  el: '.preamble-header',
+
+  events: {
+    'click .read-proposal': 'readProposal',
+    'click .write-comment': 'writeComment'
+  },
+
+  initialize: function() {
+    this.$read = $('#preamble-read');
+    this.$write = $('#preamble-write');
+    this.$readTab = this.$el.find('.read-proposal');
+    this.$writeTab = this.$el.find('.write-comment');
+
+    Backbone.on('writeSectionComment', this.writeComment, this);
```

Comment:
For consistency with the rest of the app, how about using namespaced event emitters like `CommentEvents` instead? Can we hook into the existing "comment:target" event?